packages/rgbpp/src/rgbpp/utils/transaction.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseCkbVirtualTxResult } from '@rgbpp-sdk/ckb';
+import { BtcAssetsApi } from '@rgbpp-sdk/service';
+
+export interface RgbppTxGroup {
+  ckbVirtualTxResult: BaseCkbVirtualTxResult | string;
+  btcTxHex: string;
+}
+
+export interface SentRgbppTxGroup {
+  btc: {
+    sent: boolean;
+    txId?: string;
+    error?: Error | unknown;
+  };
+  ckb: {
+    sent: boolean;
+    error?: Error | unknown;
+  };
+}
+
+export async function sendRgbppTxGroups(props: {
```

Comment:
Do you think the `sendRgbppTxGroups` method should be exposed to users? @Flouse @duanyytop

Reply:
I think it is needed for developers